src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java

```diff
@@ -73,7 +73,7 @@
   private WebElement googlePlayAppLink;
 
   private static final String PATH = "d/f/%s";
-  private static final String DEFAULT_FORUM_ID = "203236";
+  private static final String DEFAULT_FORUM_ID = "3035"; // ForumID on Fallout
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ATrailingCommentCheck)